npdarrington/instaquizzes
# Nd initial setup

### What’s this PR do?  
- Changes file structure to what is needed before starting the project
- Finalizes Typescript installation and setup with CRA
- Installs node-sass and react-router-dom
- Removes unneeded files from CRA boilerplate
- Adds normalize.scss file and imports into Index.tsx
 
### Where should the reviewer start?  
- Git pull this branch
- run `npm install` to install all dependencies on local machine
 
### How should this be manually tested?  
- run `npm start` and navigate to localhost:3000
- Ensure that no error messages show up
- Should see Instaquizzes h1 on a blank white page
 
### Any background context you want to provide?  
- Initial setup for project
 
### What are the relevant tickets?  
- closes #1
 
### Screenshots (if appropriate)  
- Initial wireframe design that the app will be designed around

<img width="775" alt="InstaQuizzes Wireframe" src="https://user-images.githubusercontent.com/59422609/98307339-c66a5900-1f82-11eb-8ec3-8fbae6acbde8.png">

### Questions: 
- N/A
